devel/googletest/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        google googletest 1.10.0 v
+revision            0
+categories          devel
+
+license             BSD
+maintainers         nomaintainer
+platforms           darwin
+
+description         Google's C++ test framework
+
+long_description    googletest is a testing framework developed by the \
+                    Testing Technology team with Google's specific \
+                    requirements and constraints in mind.
+
+github.master_sites ${github.homepage}/archive/
+distfiles           release-${version}${extract.suffix}
```

Comment:
`distfiles` should never be set directly. Both of these lines will be correctly set by the GitHub portgroup. Just tell it to grab from archives.

`github.tarball_from archives
`

Reply:
None of the `github.tarball_from` options work in this case because the tarball is not named using `${git.branch}`. Please advise:
```
        switch ${args} {
            downloads {
                github.master_sites https://github.com/downloads/${github.author}/${github.project}
            }
            releases {
                github.master_sites ${github.homepage}/releases/download/${git.branch}
            }
            archive {
                github.master_sites ${github.homepage}/archive/${git.branch}
            }
            tarball {
		github.master_sites ${github.homepage}/tarball/${git.branch}
            }
            tags {
                return -code error "the value \"tags\" is deprecated for github\
.tarball_from. Please use \"tarball\" instead."
            }
```

Reply:
nm—I see I had the tag prefix set correctly. Thanks, done.